lottoStyle.css

```
@@ -0,0 +1,466 @@
+body{
+    margin:0px;
+    background-color: #FCFCFD;
+}
+/*헤더 박스*/
+.header_b{
+    width: 1440px;
+    height: 64px;
+    border-bottom: 1px solid #4E5BA633;
+    background-color: #4E5BA6;
+}
+.header{
```

```suggestion
}

.header {
```

css 스타일 블록 사이에 개행을 하는 것과
css 선택자와 스타일 블록 사이에 공백을 두는 것은
일반적으로 권장되는 코딩 컨벤션이예요~ 그리고 이런 컨벤션에 맞춰 코드를 작성하면 일관성과 가독성을 높이는 데 도움이 되죠! 비슷한 부분들에 적용해보는 것을 추천드려요~